python_transport/wirepas_gateway/transport_service.py

```diff
@@ -621,6 +772,7 @@ def main():
     parse.add_mqtt()
     parse.add_gateway_config()
     parse.add_filtering_config()
+    parse.add_black_hole_parameters()
```

Comment:
Have you considered another name, other than black hole? I think people might not really get what the parameters are about, especially for the new ones coming into the framework. 

For example, what do you think about packet_buffer_parameters?